### PR TITLE
Make tabular menu styling consistent for arc-green

### PR DESCRIPTION
**Pull request by CirnoT**
This PR makes styling of tabular menu in forms consistent across site for arc-green and in-line with how it looks on default theme.

Before:
chrome_2020-05-23_02-39-46
chrome_2020-05-23_03-40-56

After:
chrome_2020-05-23_16-30-15
chrome_2020-05-23_16-30-04

The "After" styling was partially taken from what was previously applied only to Wiki form.

### CODE REVIEW

**Comment by silverwind** [comment]
Maybe make the tab background one shade darker? It stands out a bit too much imho.

File: web_src/less/themes/theme-arc-green.less [hunk 1]
```diff
@@ -531,15 +515,31 @@ a.ui.basic.green.label:hover {
...
 
-.repository.wiki.new .ui.container form .ui.tabular.menu {
-    border-bottom: 1px solid rgba(187, 187, 187, .6);
+.ui.tabular.menu {
+    border-bottom: 1px solid rgba(187, 187, 187, .24);
+
+    .item.active {
+        border-top: 1px solid transparent;
+        border-left: 1px solid transparent;
+        border-right: 1px solid transparent;
```
Comment (silverwind) [inline web_src/less/themes/theme-arc-green.less]: Do those need the full shorthands? If the width is already present in the base style, I suggest just setting the colors:

````css
border-top-color: transparent;
border-left-color: transparent;
border-right-color: transparent;

**Comment by CirnoT** [comment]
Changed active item to be darker and updated screenshots.

File: web_src/less/themes/theme-arc-green.less [hunk 2]
```diff
@@ -531,15 +515,31 @@ a.ui.basic.green.label:hover {
...
-        border-left: 1px solid rgba(187, 187, 187, .6);
-        border-right: 1px solid rgba(187, 187, 187, .6);
-        &:hover {
-            background: #4b5162;
+    &.navbar {
+        .item.active {
+            background: #383c4a;
+            border-left-color: transparent;
+            border-right-color: transparent;
+            border-top: 0;
```
Comment (silverwind) [inline web_src/less/themes/theme-arc-green.less]:
```suggestion
            border-top: none;
```

**Review by silverwind — APPROVED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Comment by zeripath** [comment]
Make lg-tm work

**Comment by 6543** [comment]
Pleace Backport

**Pull request merged**